src/util/coord.rs

```diff
@@ -1,5 +1,5 @@
-use std::str::FromStr;
 use super::parse_error::*;
+use std::str::FromStr;
```

Comment:
Another bit of wonky import ordering.
I prefer the python idiomatic ordering of:
```python
import builtinthing1
import buildinthing2

import externalpackage1
import externalpackage2

import internalthing1
import internalthing2
```

Reply:
Totally agree!